miden/src/cli/bundle.rs

```diff
@@ -40,19 +39,19 @@ impl BundleCmd {
                 .into_owned(),
         };
 
-        let source_manager = Arc::new(assembly::DefaultSourceManager::default());
+        let assembler = Assembler::default().with_debug_mode(true);
```

Comment:
We should make debug mode configurable (in another PR)

Reply:
Agreed! Let's create an issue for improvements to the `bundle` command. This would also include the feature described in https://github.com/0xPolygonMiden/miden-vm/pull/1445#discussion_r1713778654.

Reply:
Created https://github.com/0xPolygonMiden/miden-vm/issues/1447